subsys/net/lib/nrf_cloud/src/nrf_cloud_cell_pos.c

```diff
@@ -98,7 +98,10 @@ int nrf_cloud_cell_pos_process(const char *buf, struct nrf_cloud_cell_pos_result
 	}
 
 	err = nrf_cloud_parse_cell_pos_response(buf, result);
-	if (err < 0) {
```

Comment:
This API still returns 1 for No Location Found, can be updated to NRF_CLOUD_ERROR_DATA_NOT_FOUND instead?

Reply:
a return value of 1 indicates that this data was not a cellular positioning response.
i have updated the function comment to clarify.

`NRF_CLOUD_ERROR_DATA_NOT_FOUND` is an error value sent from nrf cloud.  if the cloud sends such an error, this function returns `-EFAULT`.